docs/versioned-plugins/inputs/beats-v4.0.1.asciidoc

```diff
@@ -229,12 +229,12 @@ The maximum TLS version allowed for the encrypted connections. The value must be
 [id="{version}-plugins-{type}s-{plugin}-tls_min_version"]
 ===== `tls_min_version` 
 
-  * Value type is <<number,number>>
+  * Value type is {logstash-ref}/configuration-file-structure.html#number[number]
   * Default value is `1`
 
 The minimum TLS version allowed for the encrypted connections. The value must be one of the following:
 1.0 for TLS 1.0, 1.1 for TLS 1.1, 1.2 for TLS 1.2
 
 
-
+[id="{version}-plugins-{type}s-{plugin}-common-options"]
```

Comment:
This problem is caused by bad asciidoc in older versions of the plugin where this line is missing. We can fix it by updating the files where this is missing.